tests/data/unicode/early_return.txt

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
```

Comment:
Everything in tests/data could go in tests/functional/u/unicode/ :) I'm going to really review after 2.12 is released. Please check other functional tests for examples.

Reply:
@Pierre-Sassoulas I will have a look. Could you approve the the workflows? I hope I fixed the compatibility issues as a starter, but as I can't see the results of the workflows (I only see error without any text) I can't be sure.

Reply:
Yeah sorry for first time contributor we need to approve each time (to prevent bitcoin miners to use github actions). You could find a typo and propose another MR so you're already trusted, but don't hesitate to ask me to approve if you need it :)